src/main/scala/com/timeout/KinesisGraphStage.scala

```diff
@@ -34,7 +38,7 @@ object KinesisGraphStage {
   * This graph stage maintains a buffer of items to push to kinesis and flushes it when full
   * The trick is that it then puts any failed items back into the buffer
   */
-class KinesisGraphStage[A : ToPutRecordsRequest](putRecords: PutRecords, streamName: String)
+class KinesisGraphStage[A : ToPutRecordsRequest](putRecords: PutRecords, streamName: String, sendingThreshold: Int)
```

Comment:
I made `sendingThreshold`. Not sure if I should expose a setter method for it as well.